server/controllers/user_controller.js

```diff
@@ -64,25 +61,21 @@ class UserController {
       });
     }
   }
+
   static signIn = async (req, res) => {
     try {
       const { email, password } = req.body;
       const login = await this.model().select('*', 'email=$1', [email]);
-
       if (login[0] && (decryptPassword(password, login[0].password))) {
-
         let token = generateAuthToken(login[0].id, login[0].is_mentor, login[0].is_admin);
```

Comment:
Create a custom method from helpers/utils that generates the token. It should accept 3 parameters (id, is_mentor, is_admin). Remove the generate token functionality and instead just call that method from here.

Reply:
I have already created this(generateAuthToken) method in helpers, it is in a file is tokens
and it takes id, is_mentor, is_admin and there I am calling it to generate token when a user signed in.

Reply:
Ooh sorry, I can see it now.